README.md

```diff
@@ -39,7 +39,7 @@ Platform Targets:
     (https://github.com/matrix-org/matrix-android-sdk2) SDKs.
 
 All code lands on the `develop` branch - `master` is only used for stable releases.
-**Please file PRs against `develop`!!**
+**Please file PRs against `development`!!**
```

Comment:
The branch is called `develop` - 
![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/ca14aad1-9dfd-44d7-9160-f880637b75ff)